### ISSUE TITLE
A11y : Cannot access Vertical slider in Query History via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46480]
**Steps to reproduce**:

Vertical slider in Query History’s first tab is impassable with the keyboard

**Actual Result**:

Degraded experience for users who rely on keyboard navigation

**Expected Result**

keyboard-users can navigate beyond first tab

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by djchazzy** [comment]
Test
